charts/signon/templates/secrets.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: external-secrets.io/v1alpha1
+kind: ExternalSecret
+metadata:
+  name: signon
+spec:
+  refreshInterval: 2m
```

Comment:
I think we should consider either omitting refreshInterval for now (default is 1h) or at least setting it to something a bit more conservative like say 30m. 2m sounds riskily frequent given than we're likely to have hundreds of secrets. (The risk isn't so much hammering Secrets Manager - which supports pretty high qps - but hammering our own etcd, which is highly undesirable.)

Reply:
Sounds reasonable, will change